pkgs/development/python-modules/fleep/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "fleep";
+  version = "1.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-yPYrJY7lNk1/bB7R8/J46ZAg/D8KYKJK0eEIRuMdEEw=";
+  };
+
+  pythonImportsCheck = [ "fleep" ];
+
```

Comment:
The package has tests we can run.

Reply:
Yes, the Pypi package does not include them, so I had to fetch it from Github instead and patch the paths on the tests. Now it is verifying them.